### ISSUE TITLE
UI of the screen should be improved accordingly.

### ISSUE THREAD

**Issue by Harshita248** [Harshita248/Vocab_Builder#2]

**Comment by samitkapoor** [comment]
@Harshita248 Can you provide some more information regarding this issue?

**Comment by Harshita248** [comment]
> @Harshita248 Can you provide some more information regarding this issue?

Yeah you can work on the UI with some animations in it and improve it.

**Comment by samitkapoor** [comment]
@Harshita248 I would love to work on this issue and I cloned the repo to see if the code is working fine.

The code is not Null sound, and can you please provide me a code that is Null sound and builds without any errors?

**Comment by Harshita248** [comment]
@samitkapoor Sure.
